README.md

```diff
@@ -36,4 +24,13 @@ Start the server
 npm run start
 ```
 
-Learn more at [http://fractal.build](http://fractal.build)
+## Front end asset management for components
+
+### CSS
+Remove the existing file `/public/styles.css`. Then, from the [fec-cms](github.com/18F/fec-cms) repo, run `npm run build-sass` and get the generated `/fec/fec/dist/fec/static/css/all-*.css` file. Then drop it in `/public/css/` folder and rename to `styles.css`.
```

Comment:
I know this isn't ideal, but this is a good first step and we can iterate toward an even better solution in the future.  Thanks for writing up the instructions!